MdePkg/Include/Protocol/BootManagerPolicy.h

```diff
@@ -31,6 +31,13 @@
     0x113B2126, 0xFC8A, 0x11E3, { 0xBD, 0x6C, 0xB8, 0xE8, 0x56, 0x2C, 0xBA, 0xFA } \
   }
 
+// MU_CHANGE START: Introduce disk connect guid
+#define EFI_BOOT_MANAGER_POLICY_STORAGE_GUID \

```

Comment:
Probably also need `gEfiBootManagerPolicyConnectStorageGuid`

Reply:
And also similar change in the dec file?

Reply:
good point